sdk/core/core-http/lib/serviceClient.ts

```diff
@@ -454,6 +469,8 @@ function createDefaultRequestPolicyFactories(authPolicyFactory: RequestPolicyFac
     factories.push(proxyPolicy(proxySettings));
   }
 
+  factories.push(logPolicy(logger.info, {}));
```

Comment:
Should this go first in the pipeline?

Reply:
.net puts it near the end. I think this should be logging after retry policy at least, so each try is logged. Other than that, I'm open to suggestions!

Reply:
Sounds fine to me!